github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+Proposed labels by section on GitHub
+====================================
+>
+Assignees
+---------------------------------------------------------------
+
+> No labels for this section - just use the alias of the individuals owning the issue. MS alias showing is preferred.
+
+Labels
+------------------------------------------------------------
+
+> It is understood that some teams will have their OWN GItHub page. Focus is on the SF page and we can build on/modify for other Team GitHub pages later once the initial model is completed.
+>
+>  
+### Label Rules for GitHub
+
+-   All tags and labels are lower-case and dash separated
+
+-   All hierarchy is whack separated and no caps.
+>  
+
+| ***State/Status*** | ***Area***               | ***Issue Type*** | ***Pri***       |
+|--------------------|--------------------------|------------------|-----------------|
+| investigating      | area-common              | question         | Pri-0,(regular) |
+| needs-more-info    | areas-communication      | enhancement      |                 |      
+| understood         | area-data-structures     | code-defect      |                 |
+| in-design          | area-developer           | test-defect      |                 |
+| in-progress        | area-diagnostics         | doc-only         |                 |
+| external           | area-documentation       |                  |                 | 
+| release-note-na    | area-engineering         |                  |                 |
+| release-note-req   | area-federation          |                  |                 |
+| duplicate          | area-hosting             |                  |                 |
+|                    | area-ktl                 |                  |                 |
+|                    | area-linux               |                  |                 |
+|                    | area-management          |                  |                 |
+|                    | area-operations          |                  |                 |
+|                    | area-reliability         |                  |                 |
+|                    | area-security            |                  |                 |
+|                    | area-setup               |                  |                 |
+|                    | area-sfrp                |                  |                 |
+|                    | area-standaloneinstaller |                  |                 |
+|                    | area-store               |                  |                 |
+|                    | area-test                |                  |                 |
```

Comment:
what is the `test` area? Shouldn't each area maintain its own tests? Should this be `area-test-infra` perhaps?